### DISCUSSION TITLE
PyBoard-D availability?

### DISCUSSION THREAD

**Discussion by hlovatt** [micropython/micropython#9307] [Hardware & Peripherals]
Is there a time frame for PyBoard-D availability?

I notice that the 722 processor is available in the 1,000 off now (out of Shanghai), but other parts might be a problem ☹️.

**Comment by dpgeorge** [comment]
Like everyone else, we are unable to source the MCUs.  In particular STM32F767IIK6 is impossible to find.

We do have a few STM32F722's left, but they are all allocated to existing orders.

All other parts we can work around / redesign, but the MCU is the show stopper at the moment.  Maybe mid next year they'll be available?  But only ST really knows the answers.

> I notice that the 722 processor is available in the 1,000 off now (out of Shanghai)

Do you have a link, what kind of prices?

**Comment by hlovatt** [comment]
Thanks for the reply. 

Will post Chinese source next week, our buyer is on holiday. 

However we have recently purchased, at a higher price, from:


Update 2: The price for STM32F722RET6 / STM32F722ZET6 has fallen to USD 35 ea from the USD 43 ea which was charged earlier. DC is 21+ so if you're interested kindly let me know. 

Best regards | Mit freundlichen Grüssen | Cordialement | Saludos | 祝万事如意|Obrigado 
 
Arizul Islam 
International Sales Representative
 
  
 
Office: 954.601.5005
Web: www.ascnow.com  
***@***.*** 
Cage Code: 4A7V3 | D&B # 36140593 | ISO 9001:2015  
Headquarters: 7880 N University Dr, Ste 100 Tamarac, FL 33321, USA 




-- Howard.

**Reply by hlovatt** [reply]
Our Chinese supplier is:

Snow Cheung
WOW SOURCING SOLUTION LIMITED
Add.: RM202. NO.7, 891 Pudong Avenue, Pudong New District.
Shanghai. P.R.China. 200120.
Tel: 0086-158-0193-2527
FAX: 0086-21-20979592
Email: [snow.wowsourcing@gmail.com](mailto:snow.wowsourcing@gmail.com)

They cost about US$20 in quantity.

If you contact Snow would say contact came from Jessica Zhang (our buyer) at Energy Renaissance so that she knows the context please.

Would this be suitable to start manufacturing PyBoard-Ds again - we would like to purchase a couple of hundred? We can do without RF parts if that would reduce cost and/or decrease supply delay.

**Reply by hlovatt** [reply]
On a further point, we could supply a couple of hundred processors if you could make the board for us. Is that something you could do?

**Reply by dpgeorge** [reply]
(we are in contact via email)
